assets/airports/kdca.json

```diff
@@ -6,673 +6,1856 @@
-                ["N39.606062600", "W77.481270900"],
-                ["N39.606713300", "W77.484101500"],
-                ["N39.607477900", "W77.486883400"],
-                ["N39.608354100", "W77.489609000"],
-                ["N39.609339300", "W77.492270600"],
-                ["N39.610430700", "W77.494860900"],
-                ["N39.611625200", "W77.497372900"],
-                ["N39.612919300", "W77.499799500"],
-                ["N39.614309400", "W77.502134400"],
-                ["N39.615791600", "W77.504371100"],
-                ["N39.617361600", "W77.506503600"],
-                ["N39.619015000", "W77.508526100"],
-                ["N39.620747300", "W77.510433400"],
-                ["N39.622553600", "W77.512220300"],
-                ["N39.624428800", "W77.513882100"],
-                ["N39.626367900", "W77.515414400"],
-                ["N39.628365500", "W77.516813300"],
-                ["N39.630416200", "W77.518075000"],
-                ["N39.632514200", "W77.519196300"],
-                ["N39.634654100", "W77.520174200"],
-                ["N39.636829900", "W77.521006300"],
-                ["N39.639036000", "W77.521690400"],
-                ["N39.641266200", "W77.522224800"],
-                ["N39.643514600", "W77.522608000"],
-                ["N39.645775200", "W77.522839000"],
-                ["N39.648042000", "W77.522917300"],
-                ["N39.650308700", "W77.522842700"],
-                ["N39.652569600", "W77.522615300"],
-                ["N39.654818400", "W77.522235600"],
-                ["N39.657049200", "W77.521704700"],
-                ["N39.659255800", "W77.521023900"],
-                ["N39.661432400", "W77.520194900"],
-                ["N39.663573200", "W77.519219700"],
-                ["N39.665672300", "W77.518101000"],
-                ["N39.667724000", "W77.516841600"],
-                ["N39.669722700", "W77.515444800"],
-                ["N39.671663100", "W77.513914200"],
-                ["N39.673539700", "W77.512253700"],
-                ["N39.675347300", "W77.510467700"],
-                ["N39.677080900", "W77.508561100"],
-                ["N39.678735700", "W77.506538700"],
-                ["N39.680307200", "W77.504406000"],
-                ["N39.681790700", "W77.502168800"],
-                ["N39.683182200", "W77.499833000"],
-                ["N39.684477600", "W77.497405000"],
-                ["N39.685673400", "W77.494891400"],
-                ["N39.686766000", "W77.492299100"],
-                ["N39.687752300", "W77.489635200"],
-                ["N39.688629400", "W77.486907100"],
-                ["N39.689394900", "W77.484122200"],
-                ["N39.690046400", "W77.481288400"],
-                ["N39.690582100", "W77.478413700"],
-                ["N39.691000300", "W77.475506000"],
-                ["N39.691300000", "W77.472573400"],
-                ["N39.691480100", "W77.469624200"],
-                ["N39.691540300", "W77.466666700"]
+                ["N38
```

Comment:
Lower limit before the upper limit on all restrictions. It doesn't break anything bur just for the sake of consistency